src/picongpu/include/particles/bremsstrahlung/SynchrotonFunctions.hpp

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright 2015 Heiko Burau, Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/cursor/Cursor.hpp"
+#include "cuSTL/cursor/navigator/PlusNavigator.hpp"
+#include "cuSTL/cursor/tools/LinearInterp1D.hpp"
+#include "cuSTL/cursor/BufferCursor.hpp"
+#include <boost/shared_ptr.hpp>
+#include <math.h>
+#include <boost/math/special_functions/bessel.hpp>
+#include <boost/array.hpp>
+#include <boost/numeric/odeint.hpp>
+#include <algorithms/math/defines/pow.hpp>
```

Comment:
is this include really necessary? or already included with "types.h" in from libPMacc.

for now, include libPMacc includes with `""` please